pkg/apis/kops/validation/aws.go

```diff
@@ -142,3 +143,13 @@ func awsValidateMixedInstancesPolicy(path *field.Path, spec *kops.MixedInstances
 
 	return errs
 }
+
+func awsValidateSSLPolicy(fieldPath *field.Path, spec *kops.LoadBalancerAccessSpec) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	if spec.SSLPolicy != nil && spec.SSLCertificate == "" {
+		allErrs = append(allErrs, field.Forbidden(fieldPath, "sslPolicy should not be specified without SSLCertificate"))
```

Comment:
given that we're only using sslPolicy on network load balancers, we should also ensure that `spec.Class` is set to `kops.LoadBalancerClassNetwork`

Reply:
Added it. Thank you.